tests/Plugins/__snapshots__/GitHub_EndToEndTest.ambr

```diff
@@ -28,7 +28,7 @@
   │ [31m│[0m                                                                                                                                                                              [31m│[0m │
   │ [31m│[0m Resolution                                                                                                                                                                   [31m│[0m │
   │ [31m│[0m ----------                                                                                                                                                                   [31m│[0m │
-  │ [31m│[0m   1) Visit 'https://github.com/gt-sse-center/RepoAuditor/settings/branches'                                                                                                  [31m│[0m │
+  │ [31m│[0m   1) Visit 'https://github.com/gt-sse-center/RepoAuditor/settings/branches'<scrubbed-space>[31m│[0m │

```

Comment:
I expected the url to be scrubbed; something like `1) Visit '<scrubbed-url>'<scrubbed-space>`

Is `ScrubDurationAndSpaces` replacing the actual url with `https://github.com/gt-sse-center/RepoAuditor?`

Reply:
Oh got it. Yes it is replacing the actual URL with the `https://github.com/gt-sse-center/RepoAuditor` since that would minimize the amount of changes in the snapshot.

I should probably rename `ScrubDurationAndSpaces` to something else after I scrub the URL as well.